index.js

```diff
@@ -0,0 +1,7 @@
+'use strict';
+
+const greet = require('./lib/greet.js');
+var name = process.argv[2];
+// console.log(process.argv);
+
+greet.sayHi(name);
```

Comment:
I see that you also have a saySeeYaLater( ) in greet.js you'll want to make sure you also invoke that function here in index.js.